docs/Installation.md

```diff
@@ -35,156 +35,129 @@ This instruction covers the following operating systems:
+> backend. If you do this on an existing system, you will wipe out the data in
+> your database.
 
-[ZONEMASTER]
-max_zonemaster_execution_time            = 300
-number_of_processes_for_frontend_testing = 20
-number_of_processes_for_batch_testing    = 20
-```
+#### 1.2.2 Instructions for PostgreSQL (CentOS)
 
-Using a database adminstrator user (called root in the example below), run the
-setup file:
+Install files to their proper locations:
 
 ```sh
-mysql --user=root --password < ./initial-mysql.sql
+cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
+sudo install -d /etc/zonemaster
+sudo install --mode=755 ./backend_config.ini-postgresql /etc/zonemaster/backend_config.ini
+sudo install --mode=755 ./zm-centos.sh-postgresql /etc/init.d/zm-centos.sh
+mkdir "$HOME/logs"
 ```
 
-This creates a database called `zonemaster`, as well as a user called
-"zonemaster" with the password "zonemaster" (as stated in the config file). This
-user has just enough permissions to run the backend software.
+Install, configure and start database engine (and Perl bindings):
 
 >
-> Note : Only run the above command during an initial installation of
-> Zonemaster Backend. If you do this on an existing system, you will wipe out the
-> data in your database.
+> At this time there is no instruction for using PostgreSQL on CentOS.
+>
+> [Sandoche! I think we should be able to support PostgreSQL!
+> Can you try to find PostgreSQL package plus perl lib?]
 >
 
- 
-If, at some point, you want to delete all traces of Zonemaster in the database,
-you can run the file `cleanup-mysql.sql` as a database administrator. Commands
-for locating and running the file are below. It removes the user and drops the
-database (obviously taking all data with it).
- 
+Initialize the database:
 
 ```sh
-cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
-mysql --user=root --password < ./cleanup-mysql.sql
+sudo -u postgres psql -f ./initial-postgres.sql
 ```
 
+> **Note:** This creates a database called `zonemaster`, as well as a user called
+> "zonemaster" with the password "zonemaster" (as stated in the config file).
+> This user has just enough permissions to run the backend software.
 
-#### 1.6.2 PostgreSQL
-
->
-> At this time there is no instruction for creating a database in PostgreSQL.
->
 
-#### 1.6.3 SQLite
+#### 1.2.3 Instructions for SQLite (CentOS)
 
 >
-> At this time there is no instruction for configuring/creating a database in PostgreSQL.
+> At this time there is no instruction for using SQLite on CentOS.
 >
 
-### 1.7 Service startup (CentOS)
+### 1.3 Service configuration and startup (CentOS)
+
+Start the service:
 
 ```sh
 sudo /etc/init.d/zm-centos.sh start
 ```
 
-### 1.8 Post-installation sanity check (CentOS)
+Check that the service has started:
+
+```sh
+sudo /etc/init.d/zm-centos.sh status
+```
+
+### 1.4 Post-installation sanity check (CentOS)

```

Comment:
The post-installation sanity check sections are repeated and could be deduplicated and moved to a common section like this: https://github.com/dotse/zonemaster-engine/blob/master/docs/Installation.md#post-installation-sanity-check.

Reply:
@mattias-p, I both agree and disagree. It *could* be moved, but I think the flow of installation would be worse. It is a important part of installation. I do not think that the duplication is such a problem. I suggest that we keep it as is. What do you say, @sandoche2k?